### DISCUSSION TITLE
How to modify config.json in 1.7.4 version?

### DISCUSSION THREAD

**Discussion by freezeskies** [Melledy/Grasscutter#2474] [Q&A] [answered]
I want to connect to the Grasscutter server on Android. The tutorial I found online says to modify the "PublicIp" in config.json, but the 1.7.4 version of config.json no longer has "PublicIp". I’m not sure how to modify it in the new version, can anyone teach me?

我想用安卓连接Grasscutter服务器，我看网上的教程说要改config.json里的"PublicIp"，但是现在在1.7.4版本我在config.json里找不到"PublicIp"了，请问我现在应该改哪些？

**Comment by NotThorny** [comment]
I don't believe that has ever been a part of config.json, it is probably referring to accessAddress.

**Reply by freezeskies** [reply]
https://github.com/Grasscutters/Grasscutter/discussions/446
![image](https://github.com/Grasscutters/Grasscutter/assets/64638710/c97bf384-23a7-46bc-8a4b-5d64aad75576)
I found the tutorial in Grasscutter's discussions, it says change PublicIp in config.json.
So I need to modify the accessAddress in config.json? Is there any other things I shoud modify?

**Reply by NotThorny** [reply, marked as answer]
accessAddress is what they mean. There are indeed two of them, and they are the only thing you change - so make sure you only change the two accessAddress spots, there's one under game and one under server in config.json.

**Reply by freezeskies** [reply]
Thank you!
I’m sorry, but I have a few more questions. Do I still need to modify the REMOTE_HOST in proxy_config.py? And my server has limited open ports, it only opens ports 7001-7300, do I need to modify the port in config.json and proxy_config.py? If so, what do I need to modify?

**Reply by NotThorny** [reply]
Yes, you need to change the remote host to the same address you changed accessAddress to.
If you have restricted ports, then change the bindPort for game and server to something that is available. You will have to change the remote port in proxy.py as well to match the port you set under server in config.json - the one that originally is 443.

**Reply by freezeskies** [reply]
OK, thank you kindly!
